src/game/common/globaldata.cpp

```diff
@@ -34,350 +34,672 @@ GlobalData *g_theWriteableGlobalData = nullptr;
 GlobalData *GlobalData::s_theOriginal = nullptr;
 const int CRC_BUFFER_SIZE = 0x10000;
 
+static const char *_terrain_lod_names[] = { "NONE",
+    "MIN",
+    "STRETCH_NO_CLOUDS",
+    "HALF_CLOUDS",
+    "NO_CLOUDS",
+    "STRETCH_CLOUDS",
+    "NO_WATER",
+    "MAX",
+    "AUTOMATIC",
+    "DISABLE",
+    nullptr };
+
 // List of keys handled in the ini
 // Class contains some variables that don't appear to be user
 // controlled.
-const FieldParse GlobalData::s_fieldParseTable[337] = { { "Windowed", &INI::Parse_Bool, nullptr, 32 }, //
-    { "XResolution", &INI::Parse_Int, nullptr, 36 }, //
-    { "YResolution", &INI::Parse_Int, nullptr, 40 }, //
-    { "MapName", &INI::Parse_AsciiString, nullptr, 8 }, //
-    { "MoveHintName", &INI::Parse_AsciiString, nullptr, 12 }, //
-    { "UseTrees", &INI::Parse_Bool, nullptr, 16 }, //
-    { "UseFPSLimit", &INI::Parse_Bool, nullptr, 20 }, //
-    { "DumpAssetUsage", &INI::Parse_Bool, nullptr, 21 }, //
-    { "FramesPerSecondLimit", &INI::Parse_Int, nullptr, 24 }, //
-    { "ChipsetType", &INI::Parse_Int, nullptr, 28 }, //
-    { "MaxShellScreens", &INI::Parse_Int, nullptr, 44 }, //
-    { "UseCloudMap", &INI::Parse_Bool, nullptr, 48 }, //
-    { "UseLightMap", &INI::Parse_Bool, nullptr, 56 }, //
-    { "BilinearTerrainTex", &INI::Parse_Bool, nullptr, 57 }, //
-    { "TrilinearTerrainTex", &INI::Parse_Bool, nullptr, 58 }, //
-    { "MultiPassTerrain", &INI::Parse_Bool, nullptr, 59 }, //
-    { "AdjustCliffTextures", &INI::Parse_Bool, nullptr, 60 }, //
-    { "Use3WayTerrainBlends", &INI::Parse_Int, nullptr, 52 }, //
-    { "StretchTerrain", &INI::Parse_Bool, nullptr, 61 }, //
-    { "UseHalfHeightMap", &INI::Parse_Bool, nullptr, 62 }, //
-    { "DrawEntireTerrain", &INI::Parse_Bool, nullptr, 63 }, //
+const FieldParse GlobalData::s_fieldParseTable[] = {
+    { "Windowed", &INI::Parse_Bool, nullptr, offsetof(GlobalData, m_windowed) }, //
```

Comment:
Can we remove the trailing comment symbols?

Reply:
Done